tests/modules/deepvariant/main.nf

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { DEEPVARIANT } from '../../../modules/deepvariant/main.nf' addParams( options: ["args": "--regions=\"chr20:10,000,000-10,010,000\" --model_type=WGS"] )
+
+workflow test_deepvariant {
+    
+    bam_tuple_ch = Channel.of([[ id:'test', single_end:false ], // meta map
+                               "https://storage.googleapis.com/deepvariant/quickstart-testdata/NA12878_S1.chr20.10_10p1mb.bam",
+                               "https://storage.googleapis.com/deepvariant/quickstart-testdata/NA12878_S1.chr20.10_10p1mb.bam.bai"])
+
+    fasta_tuple_ch = Channel.of(["https://storage.googleapis.com/deepvariant/quickstart-testdata/ucsc.hg19.chr20.unittest.fasta",
+                                 "https://storage.googleapis.com/deepvariant/quickstart-testdata/ucsc.hg19.chr20.unittest.fasta.fai"])
```

Comment:
Did you try testing with files from the [nf-core/test-datasets](https://github.com/nf-core/test-datasets/tree/modules/data/genomics/homo_sapiens) repo?

Reply:
Actually, I forgot why I didn't rely on `test-dataset` initially, but then I recalled after changing to `test_paired_end_sorted_bam` for human genome. 

I end up running into the following error message from `deepvariant` 

```
  ValueError: Reference contigs span 40001 bases but only 0 bases (0.00%) were found in common among our input files. Check that the sources were created on a common genome reference build. Contig matches were: 
  "chr22" is 40001 bp and IS MISSING. Here is a useful article about different human genome reference builds:
  https://gatkforums.broadinstitute.org/gatk/discussion/11010/human-genome-reference-builds-grch38-hg38-b37-hg19
  Please make sure the --ref input matches the build used for the input in --reads.

```

Do you suggest that I used other pairs from the `human_genome` data in `test-datasets`?

Reply:
@FriederikeHanssen Any idea about that?